tests/e2e/createWallets.spec.ts

```diff
@@ -0,0 +1,222 @@
+  await delay(1000);
+  const page = await browser.newPage();
+  await page.setViewport({ width: 1366, height: 768 });
+
+  // get extensionId
+  // https://github.com/microsoft/playwright/issues/5593#issuecomment-949813218
+  await page.goto("chrome://inspect/#extensions");
+  // https://techoverflow.net/2019/01/26/puppeteer-get-text-content-inner-html-of-an-element/
+  // TODO: check if just `page.$('...') will work because it should:
+  // https://puppeteer.github.io/puppeteer/docs/puppeteer.elementhandle
+  const url = await page.evaluate(
+    () =>
+      (
+        document.querySelector(
+          '#extensions-list div[class="url"]'
+        ) as HTMLElement
+      ).innerText
+  );
+  const [, , extensionId] = url.split("/");
+
+  const extensionOptionHtml = "welcome.html";
+  const extPage = `chrome-extension://${extensionId}/${extensionOptionHtml}`;
+
+  await page.goto(extPage);
+
+  return { page, browser };
+};
+
+test.describe("Create or connect wallets", () => {
+  test("successfully creates an Alby wallet", async () => {
+    const user = USER.SINGLE();
+
+    const { page } = await loadExtension();
+
+    // go through welcome page
+    const $document = await getDocument(page);
+
+    const startedButton = await getByText($document, "Get Started");
+    startedButton.click();
+
+    await waitFor(() => getByText($document, "Protect your wallet"));
+
+    // await findByText($document, "Protect your wallet");
+
+    // type user password and confirm password
+    const passwordField = await getByLabelText($document, "Choose a password:");
+    await passwordField.type(user.password);
+
+    const passwordConfirmationField = await getByLabelText(
+      $document,
+      "Let's confirm you typed it correct:"
+    );
+    await passwordConfirmationField.type(user.password);
+
+    // submit password form
+    const passwordFormNextButton = await getByText($document, "Next");
+    passwordFormNextButton.click();
+
+    await waitFor(() =>
+      getByText($document, "Do you have a lightning wallet?")
+    );
+
+    // click at "Create Alby Wallet"
+    const createNewWalletButton = await getByText($document, "Alby Wallet");
+    createNewWalletButton.click();
+
+    await waitFor(() => getByText($document, "Your Alby Lightning Wallet"));
+
+    // type user email
+    const emailField = await getByLabelText($document, "Email Address");
+    await emailField.type(user.email);
+
+    // type user password and confirm password
+    const walletPasswordField = await getByLabelText($document, "Password");
+    await walletPasswordField.type(user.password);
+
+    // click create a wallet button
+    const createWalletButton = await getByText($document, "Continue");
+    createWalletButton.click();
+
+    await page.waitForResponse(() => true);
+
+    await waitFor(() => getByText($document, "Your Alby account is ready."));
+
+    // submit form
+    // const nextButton = await getByText($document, "Continue");
```

Comment:
This is not working with regtest.

Reply:
Let's have a look together. I recently had issues with this and wasn't sure what the problem is.

Reply:
It's this:
![image](https://user-images.githubusercontent.com/1016218/169473510-e958833f-63fb-4c0a-8d91-527cc2471a08.png)
@kiwiidb is this still the DNS issue?

Reply:
This host should only be called from inside the cluster. I might have misconfigured this. Where is this called?

Reply:
@kiwiidb opened a ticket at DO for the DNS issue.  
Reduced the test further for now. See above comment:
```js
    // @TODO: we have an DNS issue
    // a ticket was opened at DigitalOcean
```

Reply:
I think this is wrong. we use an invalid URL here. This looks to me as a cluster internal URL. 
The internal DNS issue should have any effect here, or why does it?

Reply:
We talked. Closing this.